.github/workflows/coverage.yml

```diff
@@ -0,0 +1,41 @@
+name: Test Coverage
```

Comment:
Created using [this service](https://akx.github.io/travis-to-github-actions/) and using the deleted `.travis.yml` file as the template.